src/Windowing/OpenToolkit.Windowing.Desktop/NativeWindow.cs

```diff
@@ -757,7 +824,7 @@ public Vector2i PointToScreen(Vector2i point)
         public event EventHandler<EventArgs> Disposed;
 
         /// <inheritdoc />
-        public event EventHandler<MinimizedEventArgs> Iconified;
+        public event EventHandler<MinimizedEventArgs> Minimized;
```

Comment:
This did come up during the initial review. From what I can recall, GLFW uses the term iconified and we elected to preserve as much of the GLFW terminology as possible.

Reply:
That would require changing a ton of members around the rest of the codebase too, and I have honestly never seen the term "iconified" used to describe it.